app/server/external/CMakeLists.txt

```diff
@@ -1,22 +1,24 @@
 # This CMakeLists builds all the external libraries required for a Sonic Pi
 cmake_minimum_required(VERSION 3.12)
+project(SonicPiServerExternal
+    LANGUAGES CXX C
+    VERSION 1.0.0
+)
 
 # Osmid
 include(ExternalProject)
 ExternalProject_Add(osmid
     PREFIX osmid-prefix
-    SOURCE_DIR ${CMAKE_CURRENT_LIST_DIR}/osmid
-    INSTALL_COMMAND "" 
+    SOURCE_DIR "${CMAKE_CURRENT_LIST_DIR}/osmid"
+    INSTALL_DIR "${CMAKE_CURRENT_LIST_DIR}/../native/osmid"
     BUILD_COMMAND ${CMAKE_COMMAND} --build . --config Release
+    INSTALL_COMMAND ${CMAKE_COMMAND} -E copy "<BINARY_DIR>/m2o" "<BINARY_DIR>/o2m" "<INSTALL_DIR>"
 )
```

Comment:
I think using <BINARY_DIR> won't work so well on windows.  It has mulitple configs.  Need to check it is OK.

Reply:
Ah, ok.

Reply:
It seems like it worked on my few quick tests on Windows. Hopefully it'll be ok to use :)